example/worker-machine.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  name: worker-machine
+  namespace: default
+  labels:
+    machine.openshift.io/cluster-api-machineset: worker-machine
+    machine.openshift.io/cluster-api-cluster: equinixmetal-actuator-k8s
+spec:
+  providerSpec:
+    value:
+      apiVersion: equinixmetalproviderconfig.openshift.io/v1beta1
```

Comment:
I believe this should be equinixmetal.machine.openshift.io/v1beta1

Reply:
Fixed, thanks!